src/freenas/root/.zshrc

```diff
@@ -1,9 +1,14 @@
 
 HISTFILE=~/.zsh-histfile
 SAVEHIST=1000
+PROMPT="%n@%m# "
```

Comment:
Can we also include `pwd` then? Like `%20<...<%~%<<`

Reply:
Cuts path from the start if it is too long. Probably `%40<...<%~%<<` would be a better width.